### PR TITLE
identified user behaviour and create method in personal information s…

### PR DESCRIPTION
**Pull request by anupama-lakshmi-improve10x**
…creen

### CODE REVIEW

File: app/src/main/java/com/improve10x/learningplatform/MainActivity.java [hunk 1]
```diff
@@ -11,4 +13,15 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
     }
+    private void handleSaveClick() {
+    }
+
+    private void getImage() {
+    }
```
Comment (sriraghu-arigela-improve10x) [inline app/src/main/java/com/improve10x/learningplatform/MainActivity.java]: give space

File: app/src/main/java/com/improve10x/learningplatform/MainActivity.java [hunk 2]
```diff
@@ -11,4 +13,15 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
     }
```
Comment (sriraghu-arigela-improve10x) [inline app/src/main/java/com/improve10x/learningplatform/MainActivity.java]: give space

File: app/src/main/java/com/improve10x/learningplatform/MainActivity.java [hunk 3]
```diff
@@ -11,4 +13,15 @@ protected void onCreate(Bundle savedInstanceState) {
...
+    private void handleSaveClick() {
```
Comment (rajesh-pujari-improve10x) [inline app/src/main/java/com/improve10x/learningplatform/MainActivity.java:16]: you did this all work in main activity, plz check it once

**Review by sriraghu-arigela-improve10x — APPROVED** [review]

File: app/src/main/java/com/improve10x/learningplatform/PersonalInformationFragment.java [hunk 4]
```diff
@@ -15,4 +15,28 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         return inflater.inflate(R.layout.fragment_personal_information, container, false);
     }
+
+    private void handleSaveClick() {
```
Comment (rajesh-pujari-improve10x) [inline app/src/main/java/com/improve10x/learningplatform/PersonalInformationFragment.java:19]: give handleSaveChanges()

File: app/src/main/java/com/improve10x/learningplatform/PersonalInformationFragment.java [hunk 5]
```diff
@@ -15,4 +15,28 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+    }
+
+    private void getImage() {
+    }
+
+    private void getNameText() {
+    }
+
+    private void handleBackClick() {
```
[the remark this answers is not in the document]
Comment (anupama-lakshmi-improve10x) [reply app/src/main/java/com/improve10x/learningplatform/PersonalInformationFragment.java:28]: it's not a button.. so I used click

[the remark this answers is not in the document]
Comment (rajesh-pujari-improve10x) [reply app/src/main/java/com/improve10x/learningplatform/PersonalInformationFragment.java:28]: ok

**Review by rajesh-pujari-improve10x — APPROVED** [review]

**Review by rajesh-pujari-improve10x — APPROVED** [review]

**Review by sriraghu-arigela-improve10x — APPROVED** [review]

**Review by manoj-improve10x — APPROVED** [review]

**Pull request merged**